.eslintrc.js

```diff
@@ -54,14 +53,8 @@ module.exports = {
     // no need for error
     'pii/no-phone-number': 'warn',
     'padding-line-between-statements': ['warn', { blankLine: 'always', prev: '*', next: 'return' }],
-    'import/extensions': 'off',
-    'promise/catch-or-return': 'off',
-    '@typescript-eslint/no-floating-promises': 'off',
-    'promise/always-return': 'off',
-    'unicorn/no-array-reduce': 'off',
-    'no-restricted-syntax': 'off',
-    'no-continue': 'off',
+    '@typescript-eslint/no-floating-promises': 'warn',
     'const-case/uppercase': 'off',
-    'prettier/prettier': 'error',
+    'import/extensions': 'off',
   },
-}
+}
```

Comment:
Turn on this setting in VSCode to keep newlines at the end :) 
```
  "files.insertFinalNewline": true,
```

More in docs: https://bratislava.github.io/onboarding/editor-setup#setup-formatting-and-linting